packages/utils/src/ReducerRegistry/ReducerRegistry.ts

```diff
@@ -41,12 +52,12 @@ export class ReducerRegistry {
    *
    * @param newReducers the object of new reducers.
    */
-  register(newReducers) {
+  register(newReducers: Record<string, never>) {
```

Comment:
Can you use `any` instead of `never`?

```TS
newReducers: Record<string, any>
```

After this change you might be able to remove the type casting bellow.

Reply:
Done, Cap'n 💯